server-embedded/src/main/java/org/exoplatform/chat/server/ChatServer.java

```diff
@@ -862,10 +867,16 @@ public ChatServer()
     if (!tokenService.hasUserWithToken(user, token)) {
       return Response.notFound("Petit malin !");
     }
-    
-    chatService.setRoomMeetingStatus(room, Boolean.parseBoolean(start), startTime);
-
-    return Response.ok("Updated.");
+  
+    //only member of a room can updateRoomMeetingStatus
+    List<UserBean> users = userService.getUsers(room);
+    users=users.stream().filter(userBean -> userBean.getName().equals(user)).collect(Collectors.toList());
```

Comment:
I think it's better to make `org.exoplatform.chat.services.ChatServiceImpl.isMemberOfRoom(String, String)` public and reuse it here, WDYT ?